bin/rector.php

```diff
@@ -21,15 +20,19 @@
 
 define('__RECTOR_RUNNING__', true);
 
+
 // Require Composer autoload.php
 $autoloadIncluder = new AutoloadIncluder();
 $autoloadIncluder->includeDependencyOrRepositoryVendorAutoloadIfExists();
 
-if (file_exists(__DIR__ . '/../vendor/scoper-autoload.php')) {
-    // make local php-parser a priority to avoid conflict
+// load local php-parser only in prefixed version or development repository
+$isDevelopmentRepository = file_exists(__DIR__ . '/../phpstan-for-rector.neon');
+if (file_exists(__DIR__ . '/../vendor/scoper-autoload.php') || $isDevelopmentRepository) {
     require_once __DIR__ . '/../preload.php';
 }
```

Comment:
@samsonasik The local repository detection is needed too, as we have `phpstan.phar` autoload that is loaded first and creates conflict with our `php-parser`

Reply:
@TomasVotruba thank you 👍